python/cudf/cudf/core/udf/lowering.py

```diff
@@ -0,0 +1,114 @@
+from numba.cuda.cudaimpl import (
+    lower as cuda_lower,
+    registry as cuda_lowering_registry,
+)
+from numba.core.typing import signature as nb_signature
+from cudf.core.udf.typing import MaskedType, NAType
+from numba.core import cgutils
+from numba.cuda.cudaimpl import registry as cuda_impl_registry
+import operator
+from numba.extending import types
+
+@cuda_lowering_registry.lower_constant(NAType)
+def constant_dummy(context, builder, ty, pyval):
```

Comment:
I'd be inclined to name the `constant_na` so that if it appears in a traceback or error message for some reason, it'll be a little clearer what it was implementing (as opposed to the name suggesting what it returns instead).